README.md

```diff
@@ -1 +1,36 @@
-# Enumerable
\ No newline at end of file
+# Enumerable
+- Use [enumerable.rb](./enumerable.rb) to use any? ,  all? and filter functionality.
+
+
```

Comment:
It will be nice to add other main headings with some brief descriptions such as :

- Getting started
- Set up
- Install
- Usage
- Run tests